File: Java/Lista 3/L03EX02/src/Regular.java

public class Regular extends Conta {

	public Regular(String nome, String nun_conta, Double saldo) {
		super(nome, nun_conta, saldo);
	}
	
	
	public Double getTarRegular() {
		return (this.getSaldo() * 0.01);
	}
	
    public void mostraTarRegular() {
    	System.out.println("Tarifa: R$ " + this.getTarRegular());
    }
    
    
    public void mostraRegular() {
    	System.out.println("Numero da Conta........: " + this.getNun_conta());
    	System.out.println("Nome...................: " + this.getNome());
    	System.out.println("Saldo..................: R$ " + this.getSaldo());
    	System.out.println("Tarifa Regular (1%)....: R$ " + this.getTarRegular());
     	
    }

	
}

File: Swift/L03EX02/L03EX02/Regular.swift
// Regular.swift - L03EX02


import Foundation

public class Regular: Conta{
    
    
    public func getTarRegular() -> Double{
        return (self.getSaldo() * 0.01)
    }
    
    public func printTarRegular(){
        print("A tarifa Regular (1%) do(a) \(self.getNome()): R$ \(self.getTarRegular())\n")
    }
    
    public func mostraClienteRegular(){
        print("Conta Número: \(self.getNumConta())")
        print("Nome Cliente: \(self.getNome())")
        print("saldo: R$ \(self.getSaldo())")
        print("A tarifa Regular (2%): R$ \(self.getTarRegular())\n")
    }
    
}
